src/ui/Forms/FixCommonErrors.cs

```diff
@@ -681,18 +681,7 @@ public void LogStatus(string sender, string message)
                 }
             }
         }
-
-        public object GetCustomCallbackData(IFixCommonError sender, object input)
```

Comment:
This method doesn't belong here!

We should only methods/props that is common to all the fix-common-errors logics